libsepol/src/policydb_validate.c

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
The commit still does an unrelated whitespace change, which is often considered to be a bad practice. It's better to leave it as it is and submit the empty line removal as a separate patch.

Reply:
Oh, thought that the spaces and also the line was added by me by accident. Will remove it from the patch.